app/stateMachines/GetStartedStateMachine.js

```diff
@@ -311,78 +310,29 @@ export const getStartedMachine = Machine({
         }
       }
     },
-    // XXX
-    // make a machine for final configurations, similar to how we create
-    // the create wallet machine, so we can set mix accounts sync vsp and
-    // any other config which may come.
-    settingMixedAccount: {
-      onEntry: "isAtSettingAccount",
-      initial: "settingMixedAccount",
-      states: {
-        settingMixedAccount: {}
-      },
-      on: {
-        CONTINUE: "syncVSPTickets"
-      }
-    },
-    syncVSPTickets: {
-      onEntry: "isAtSyncingVSPTickets",
-      initial: "syncingVSPTickets",
-      states: {
-        syncingVSPTickets: {}
-      },
-      on: {
-        FINISH: "goToHomeView",
-        CONTINUE: "processingManagedTickets"
-      }
-    },
-    processingManagedTickets: {
-      onEntry: "isAtProcessingManagedTickets",
-      initial: "processingManagedTickets",
-      states: {
-        processingManagedTickets: {}
-      },
-      on: {
-        BACK: "processingUnmanagedTickets",
-        CONTINUE: "processingUnmanagedTickets",
-        ERROR: {
-          target: "processingManagedTickets",
-          actions: assign({
-            error: (context, event) => event.error && event.error
-          })
-        }
-      }
-    },
-    processingUnmanagedTickets: {
-      onEntry: "isAtProcessingUnmanagedTickets",
-      initial: "processingUnmanagedTickets",
-      states: {
-        processingUnmanagedTickets: {}
-      },
-      on: {
-        CONTINUE: "goToHomeView",
-        BACK: "goToHomeView",
-        ERROR: {
-          target: "processingUnmanagedTickets",
-          actions: assign({
-            error: (context, event) => event.error && event.error
-          })
-        }
-      }
-    },
-    isLoadingConfig: {
-      onEntry: "isAtLoadingConfig",
-      initial: "isLoadingConfig",
+    settingUpWallet: {
+      initial: "settingUpWallet",
       states: {
-        isLoadingConfig: {}
-      },
-      on: {
-        FINISH: "goToHomeView",
-        CONTINUE: "processingUnmanagedTickets",
-        ERROR: {
-          target: "processingManagedTickets",
-          actions: assign({
-            error: (context, event) => event.error && event.error
+        settingUpWallet: {
+          entry: assign({
+            settingUpWalletRef: (ctx) => {
+              let spawnedMachine;
+              // spawn a new actor machine so we can comunicate with the
+              // getStartedMachine.
+
+              // source: https://xstate.js.org/docs/guides/actors.html#spawning-machines
+              try {
+                spawnedMachine = spawn(
+                  SetupWalletConfigMachine.withContext({
+                    selectedWallet: ctx.selectedWallet,
+                    isCreateNewWallet: ctx.isCreateNewWallet
+                  })
+                );
+              } catch (e) {
+                console.log(e);
```

Comment:
is this console.log necessary / helpful ?

Reply:
it isn't supposed to get there, but I think it might be good, loggin in case